components/webui/package.json

```
@@ -2,16 +2,15 @@
   "name": "webui",
   "version": "0.1.0",
   "description": "",
+  "workspaces": [
+    "common",
+    "client",
+    "server"
+  ],
   "scripts": {
-    "client:lint:check": "cd client && npm run lint:check",
-    "client:lint:fix": "cd client && npm run lint:fix",
-    "client:start": "cd client && npm start",
-    "init": "npm clean-install && (cd client && npm clean-install) && (cd server && npm clean-install)",
-    "lint:check": "npm run client:lint:check && npm run server:lint:check",
-    "lint:fix": "npm run client:lint:fix && npm run server:lint:fix",
-    "server:lint:check": "cd server && npm run lint:check",
-    "server:lint:fix": "cd server && npm run lint:fix",
-    "server:start": "cd server && npm start",
+    "init": "npm clean-install --workspaces",
+    "lint:check": "npm --workspace common run build && npm run lint:check --workspaces",
+    "lint:fix": "npm --workspace common run build && npm run lint:fix --workspaces",
```

otherwise, to run the client or server, developers will have to manually build first
```suggestion
    "init": "npm clean-install --workspaces && npm --workspace common run build",
    "lint:check": "npm run lint:check --workspaces",
    "lint:fix": "npm run lint:fix --workspaces",
```